metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -680,10 +695,10 @@ def get_custom_properties(node: DBTNode) -> Dict[str, str]:
 
 
 def _get_dbt_cte_names(name: str, target_platform: str) -> List[str]:
-    # Match the dbt CTE naming scheme:
-    # The default is defined here https://github.com/dbt-labs/dbt-core/blob/4122f6c308c88be4a24c1ea490802239a4c1abb8/core/dbt/adapters/base/relation.py#L222
-    # However, since this PR https://github.com/dbt-labs/dbt-core/pull/2712, it's also possible
-    # for adapters to override this default. Only a handful actually do though:
+    # Match the dbt CTE naming scheme: The default is defined here
+    # https://github.com/dbt-labs/dbt-core/blob/4122f6c308c88be4a24c1ea490802239a4c1abb8/core/dbt/adapters/base
+    # /relation.py#L222 However, since this PR https://github.com/dbt-labs/dbt-core/pull/2712, it's also possible for
```

Comment:
undo this comment change - can't be breaking up links

overall - why are all these comments and strings being reformatted?

Reply:
they are going over the 80 characters limit and hard to follow as long line. I will revert this

Reply:
reverted the changes.